king/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set name = "KING" %}
+{% set version = "0.0.0" %}
```

Comment:
I see a different version number on the [website](http://people.virginia.edu/~wc9c/KING/Download.htm)